packages/fxa-shared/metrics/amplitude.js

```diff
@@ -442,3 +443,18 @@ function mapLocation(location) {
     };
   }
 }
+
+function estimateTime(times) {
+  const skew = times.received - times.sent;
+  return times.start + times.offset + skew;
+}
+
+function mapTime(data, receivedTime) {
```

Comment:
The call site passes in `data`, which does not have the `offset`.   `offset` is a property in each event.